src/ApolloClient/client.js

```diff
@@ -1,8 +1,11 @@
 import {ApolloClient, InMemoryCache} from '@apollo/client';
-const client = new ApolloClient
-({
-  uri: 'https://fff-be-2e7913919a6b.herokuapp.com/graphql',
-  // uri: 'http://127.0.0.1:6969/graphql',
+
+const uri = process.env.NODE_ENV === 'production'
+  ? 'https://fff-be-2e7913919a6b.herokuapp.com/graphql'
+  : 'http://127.0.0.1:6969/graphql';
+
```

Comment:
This code block checks the current environment (NODE_ENV). If the application is running in production mode, it uses the production GraphQL API endpoint hosted on Heroku (https://fff-be-2e7913919a6b.herokuapp.com/graphql). If it's in development mode, it uses the local GraphQL API endpoint (http://127.0.0.1:6969/graphql).